localstack/utils/aws/aws_responses.py

```diff
@@ -52,6 +52,7 @@ def flask_error_response_json(
     result = {
         "Type": "User" if code < 500 else "Server",
         "message": msg,
+        "Message": msg,
```

Comment:
@dominikschubert what you think about this? Some of the Exceptions expect the response to have one or the other one, or both `message` and `Message`. 

Should I just create a new function or return directly the error response without using this method?